src/main/java/org/openpnp/machine/reference/ReferencePnpJobProcessor.java

```diff
@@ -279,6 +281,12 @@ protected void doPreFlight() throws Exception {
                 continue;
             }
             
+            // Check if board z-height is still 0 
```

Comment:
The reference classes cannot refer to the GUI classes as this breaks tests and locks the job processor to this specific UI. Suggest just removing this block entirely. If you really feel a popup warning is necessary it can be done in the JobPanel on job start.

Reply:
done